analytics/tools/patch_eval_format/main.py

```diff
@@ -0,0 +1,36 @@
+"""
+# Motivation
+
+Patches old logfiles to the new evaluation logfile format with batched evaluations.
+"""
+
+from pathlib import Path
+from typing import Annotated
+
+import typer
+
+from analytics.app.data.load import list_pipelines
+from analytics.tools.patch_eval_format.patch_eval import patch_logfile
+
+
+def main(
+    logs_directory: Annotated[Path, typer.Argument(help="Path to read the pipelines in from")],
+    patched_log_dir: Annotated[Path, typer.Argument(help="Path to output the patched pipelines to")],
+) -> None:
+    # find the groups of equivalent pipelines via the .name file
+
+    pipelines = list_pipelines(logs_directory)
+
+    for pipeline_id, (pipe_name, pipeline_path) in pipelines.items():
+        logfile = logs_directory / pipeline_path / "pipeline.log"
+        if "rho_loss" in pipe_name:
```

Comment:
why is `rho_loss` special?

Reply:
The problem is that the rho loss config changed since running the old pipelines. Hence we cannot parse the old files by just porting the evaluation stuff

Reply:
I see. the old rho config is not useful anyway